.github/pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+ <!--  Thanks for sending a pull request!  Here are some tips for you:
+
+    1. If this is your first time, please read our contributor guidelines: https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING.md
+    2. Discuss your changes before you start working on them. You can open a new issue in the [Sail Operator GitHub repository](https://github.com/istio-ecosystem/sail-operator/issues) or start a discussion in the [Sail Operator Discussion](https://github.com/istio-ecosystem/sail-operator/discussions). By this way, you can get feedback from the community and ensure that your changes are aligned with the project goals.
+    3. If the PR is unfinished, make is as draft.
```

Comment:
Hey, maybe rephrase `If the PR is unfinished, mark it as a draft.`

Reply:
Done